cleanlab/object_detection/summary.py

```diff
@@ -265,6 +265,9 @@ def plot_class_size_distributions(
     class_to_show: optional
         The number of classes to show in the plots. Classes over `class_to_show` are hidden. If this argument is provided, then the classes are sorted by the number of instances in the dataset.
         Defaults to `MAX_CLASS_TO_SHOW` which is set to 10.
+
+    **kwargs:
```

Comment:
```suggestion
    kwargs:
```

Reply:
I'm basing this off this existing example:
https://github.com/cleanlab/cleanlab/blob/master/cleanlab/token_classification/filter.py

I don't 100% recall if this ^ is the desired documentation format, but it looks good to me.